Files/App.xaml

```diff
@@ -166,6 +166,7 @@
                             <Color x:Key="LayerFillColorDefault">#11000000</Color>
                         </ResourceDictionary>
                     </ResourceDictionary.ThemeDictionaries>
+                    <x:Double x:Key="SettingsBlockControlDefaultWidth">385</x:Double>
```

Comment:
Should this be 396?

Reply:
I just took a value that fitted, there was no deeper though process than "Yup this works".

Reply:
We try to make sure most values are divisible by 4 so I think we should try to match that.

Reply:
Alrighty updated the values.